src/develop/masks/masks.c

```diff
@@ -1245,6 +1245,12 @@ int dt_masks_events_mouse_leave(struct dt_iop_module_t *module)
   if(darktable.develop->form_gui)
   {
     dt_masks_form_gui_t *gui = darktable.develop->form_gui;
+
+    // if masks are being created or edited don't reset the position
```

Comment:
That's it, so you've disable the move at the center when moving the mouse out of the darkroom. I have done just the opposite in liquify. I find it better to have the mask at the center instead of staying on the border and sometime not very visible.

Reply:
I've done this so when the mask or a node/segment is dragging and the mouse get out of the darkroom it doesn't move to the left-top corner.
In creation, I like better the preview to stay at the edge instead of jump to the middle, but that's a matter of taste. To change it back I think that the only thing needed is to remove the gui->creation from the if(). Let me know if you want that.

Reply:
I think consistency if very important. I must say that I have no string opinion on this. The only thing is that we must choose one option had have the masks, the retouch and the liquify works the same. Any opinion on that? We need to agree on one behavior :)

Reply:
If you really like better to have the preview stays on the border I'm ok with that and will change liquify accordingly. Just to be sure we agree on this.

Reply:
I agree that consistency is important. And I do like this behaviour better. When I move the mouse outside the darkroom area my eyes are on the destination and I find distracting to see with the corner of my eyes the preview jumping to the center.

Reply:
Done.